samples/IdealPostcodes/apiDefinition.swagger.json

```diff
@@ -8,221 +8,226 @@
-          "type": "string"
+          "type": "string",
+          "title": "Small User Organisation Indicator",
+          "x-ms-visibility": "advanced"
         },
         "sub_building_name": {
           "description": "When a premise is split into individual units such as flats, apartments or business units. Cannot be present without either building_name or building_number. E.g. Flat 1, A, 10B",
           "maxLength": 30,
-          "type": "string"
+          "type": "string",
+          "title": "Sub Building Name",
+          "x-ms-visibility": "advanced"
         },
         "thoroughfare": {
           "description": "Also known as the street or road name. In general each Thoroughfare Name will have a separate Postcode. Longer Thoroughfares with high number ranges often have multiple Postcodes covering the entire length of the road, with breaks at suitable points e.g. junctions or natural breaks in the road.",
           "maxLength": 80,
-          "type": "string"
+          "type": "string",
+          "title": "Thoroughfare",
+          "x-ms-visibility": "important"
         },
         "traditional_county": {
           "description": "Traditional counties are provided by the Association of British Counties. It\u2019s historical data, and can date from the 1800s. Data source: Royal Mail. May be empty.",
-          "type": "string"
+          "type": "string",
+          "title": "Traditional County",
+          "x-ms-visibility": "advanced"
         },
         "udprn": {
           "description": "UDPRN stands for \u2018Unique Delivery Point Reference Number\u2019. Royal Mail assigns a unique UDPRN code for each premise on PAF. Simple, unique reference number for each Delivery Point. Unlikely to be reused when an address expires. Up to 8-digit numeric code. A new UDPRN is automatically assigned to each new Delivery Point added to PAF.",
           "format": "int32",
-          "type": "integer"
+          "type": "integer",
+          "title": "UDPRN",
+          "x-ms-visibility": "advanced"
         },
         "umprn": {
           "description": "A small minority of individual premises (as identified by a UDPRN) may have multiple occupants behind the same letterbox. These are known as Multiple Residence occupants and can be queried via the Multiple Residence dataset. Simple, unique reference number for each Multiple Residence occupant. 8-character numeric code. Note: this will be an empty string when not used for legacy reasons",
-          "type": "string"
+          "type": "string",
+          "title": "UMPRN",
+          "x-ms-visibility": "advanced"
         },
         "ward": {
           "description": "The current administrative/electoral area to which the postcode has been assigned. May be empty for a small number of addresses. Data source: ONS",
-          "type": "string"
-        }
-      },
-      "title": "Address",
-      "type": "object"
-    },
-    "AddressAutocompleteHit": {
```

Comment:
It was simply to streamline the connector down to a handful of useful actions

- Address Search
- Get Address by UDPRN
- Get Addresses by Post Code
- Lookup UDPRN
- Lookup UDPRM

Reply:
@garrytrinder In future, we would like to publish and update connectors directly from the open source repo. Instead of publishing a subset, is it possible to publish the full set of actions for open source?